src/bin/pgcopydb/cli_dump.c

```diff
@@ -33,11 +33,12 @@ static CommandLine dump_schema_command =
 		"schema",
 		"Dump source database schema as custom files in work directory",
 		" --source <URI> ",
-		"  --source          Postgres URI to the source database\n"
-		"  --target          Directory where to save the dump files\n"
-		"  --dir             Work directory to use\n"
-		"  --skip-extensions Skip restoring extensions\n" \
-		"  --snapshot        Use snapshot obtained with pg_export_snapshot\n",
+		"  --source				Postgres URI to the source database\n"
+		"  --target          	Directory where to save the dump files\n"
+		"  --dir				Work directory to use\n"
+		"  --skip-extensions	Skip restoring extensions\n" \
+		"  --filters <filename> Use the filters defined in <filename>\n"
+		"  --snapshot			Use snapshot obtained with pg_export_snapshot\n",
```

Comment:
That's here: use only spaces please.

Reply:
@dimitri I had used `make update-docs` to update the docs.

Reply:
The alignment issue is with using TABs in the code instead of using only spaces.